### DISCUSSION TITLE
Skin file name vs Name in skin.ini

### DISCUSSION THREAD

**Discussion by kaimuu** [ppy/osu#21859] [Ideas] [answered]
Currently all skins are shown with their name and author from the skin.ini file, while this is a nice idea and it should probably stay to some degree this does add more steps to sort skins ingame by giving them different names (I basically have no idea what 80% of my skins even are since I do have around 1200 of them but named most of them in a way where I know what it is/in a way where they are in a certain spot in the list) since you would have to open the .osk with 7zip or WinRAR and change the name inside of there instead of just the simple step of renaming the file name.

This is especially an issue because either way if you want to change a name of a skin you already have ingame you basically have to export it, change the name the way I described earlier and then import it back while also deleting the earlier version. Being able to rename the skin from inside the game would also help this a lot but depending on the implementation might still take a lot longer than it would currently.

**Comment by frenzibyte** [comment, marked as answer]
This will be possible as part of a skin manager overlay in https://github.com/ppy/osu/issues/18883
